lib/config/options/index.ts

```diff
@@ -229,6 +229,16 @@ const options: RenovateOptions[] = [
     default: 'disabled',
     experimental: true,
   },
+  {
+    name: 'repositoryCacheType',
+    description:
+      'Set the type of renovate repository cache if repositoryCache is enabled.',
```

Comment:
repositoryCache allowedValues: disabled, enabled, reset
```suggestion
      'Set the type of renovate repository cache if repositoryCache is not disabled.',
```

Reply:
we should set to enum like and add allowed values

Reply:
we have additional `repositoryCacheType`, which defaults to 'local' but it is intended to be 
`'local' | 's3' | 'redis'`
if thats what you mean

Reply:
didnt want to change existing behavior + config options.